README.md

```diff
@@ -208,59 +274,15 @@ A device attribute MUST be one of these:
     <td>Yes</td>
     <td>No</td>
   </tr>
-  <tr>
-    <td>$fw/name</td>
-    <td>Device → Controller</td>
-    <td>Name of the firmware running on the device. Allowed characters are the same as the device ID</td>
-    <td>Yes</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td>$fw/version</td>
-    <td>Device → Controller</td>
-    <td>Version of the firmware running on the device</td>
-    <td>Yes</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td>$nodes</td>
-    <td>Device → Controller</td>
-    <td>
-      Nodes the device exposes, with format <code>id</code> separated by a <code>,</code> if there are multiple nodes.
-    </td>
-    <td>Yes</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td>$implementation</td>
-    <td>Device → Controller</td>
-    <td>An identifier for the Homie implementation (example <code>esp8266</code>)</td>
-    <td>Yes</td>
-    <td>Yes</td>
-  </tr>
-  <tr>
-    <td>$implementation/#</td>
-    <td>Controller → Device or Device → Controller</td>
-    <td>You can use any subtopics of <code>$implementation</code> for anything related to your specific Homie implementation.</td>
-    <td>Yes or No, depending of your implementation</td>
-    <td>No</td>
-  </tr>
 </table>
 
-For example, a device with an ID of `686f6d6965` with an `outdoor-probe` and a `windsensor` node would send:
+For example, a device with an ID of `686f6d6965` with `$stats/$interval` value "60" may send every 60seconds its current values:
```

Comment:
60seconds
may?
Suggestion: "is supposed to send it's current values every 60 seconds."